### DISCUSSION TITLE
When to use vue-dapp

### DISCUSSION THREAD

**Discussion by javaspeak** [vu3th/vue-dapp#142] [General]
I have seen:

       https://docs.walletconnect.com/2.0/web/web3modal/html/wagmi/installation

The web3modal docs say that it can be used by any framework.

Am missing a bit of context about how vue-dapp enhances or differs from the above link.

Can anyone enlighten me.  Thanks for bearing with this newbee question.

**Comment by re2005** [comment]
Hi @javaspeak `Vue-dapp` uses  `@walletconnect/ethereum-provider` to provide connection with WalletConnect.

https://github.com/vu3th/vue-dapp/blob/main/src/connectors/walletConnect.ts#L13

Not sure what is your question about ?

`Vue-dapp` is a VUE plugin to enhance developer and user experience.
